Week-2/Assignment-1/index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html />
+<html>
+
+<head>
```

Comment:
Indentation is missing.

Reply:
Added Indentation with prettier code formatter